Jenkinsfile

```diff
@@ -10,34 +10,56 @@
 parallel (
     'intel': {
         node("Yocto") {
-            checkout scm
-            def manifests = load "ci-scripts/yocto.groovy"
-            manifests.buildManifest("intel", "core-image-pelux-minimal")
+            boolean only_qemu = env.QEMU == "true"
```

Comment:
That is a really weird way of implementing that by copy&pasting it that many times.

I would not try to do a special thing for qemu, but instead abstract it away so it is easy to extend for example you could create this map programatically before the parallel() call. And there you could have a list of the jobs which are default (which would be all of them) and then prepare so one can manipulate this list to remove some of them, or overwrite it with only qemu in it on demand.

This way you would make it possible to just run the rpi build ord only the qemu and intel-qtauto, or any other combination without every time adding special if clauses.

Reply:
I guess you mean this as an improvement to the Jenkinsfile, so not actually within the scope of adding smoke tests. It does make sense to me to try to get that to work.

Reply:
I can remove the qemu flag. I just wanted to use it for being able to run from Jenkins in a simple way without hard-coding.

Reply:
Sorry. I misunderstood your comment. Yes, I agree this is not a nice way of doing this.